### ISSUE TITLE
Browserlify compatibility

### ISSUE THREAD

**Issue by Reggino** [logentries/le_js#55]
Following up issue #25 and issue #33 

```le_js``` still isn't compatible with Browserify due this line: 

https://github.com/logentries/le_js/blob/master/src/le.js#L31

```this``` is not (always?) ```window``` in the case of Browserify, as seen https://github.com/substack/node-browserify/issues/372 and http://stackoverflow.com/questions/20724838/passing-in-window-to-this-in-browserify

**Comment by bathos** [comment]
Indeed. The simplest solution is to eliminate the `window` argument in the "factory", and allow references to `window` within it to be direct. Unless there's some issue with doing so when it comes to the other module systems that the boilerplate attempts to cover?

**Comment by Reggino** [comment]
@bathos, right that is how others seem to cover it... (e.g. https://github.com/es-shims/es5-shim/blob/master/es5-shim.js#L30)
